index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Portfolio Website</title>
+</head>
+
+<body>
+  <div class="header">
+    <div class="placeHolder">
+      <h1 class="head">Welcome</h1>
+    </div>
+    <div class="nbtn1">
+
+      <div class="v1"></div>
+      <div class="v2"></div>
+      <div class="v3"></div>
+
+    </div>
+  </div>
+
+  <div class="container">
+
+
+
+    <!--Main part-->
+
+    <div class="Headline">
+      <div class="Primary">
+        <h3 style="color: #172B4D;">Hey there. I'm Faisal</h3>
```

Comment:
- [ ] The main title (e.g., "Hey there. I'm Faisal") should be an `h1` tag. * If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. This is the title that usually shows up in google searches, it is mostly possible because the `h1` tag is used.
- [ ] Please apply all styles in your `style.css` file as in-line style is greatly discouraged.